tests/components/zwave_js/test_light.py

```diff
@@ -597,125 +615,67 @@ async def test_black_is_off(
-                    "green": 255,
-                    "blue": 0,
-                },
-                "propertyName": "currentColor",
+                "commandClassName": "Binary Switch",
+                "commandClass": 37,
+                "endpoint": 1,
+                "property": "currentValue",
+                "newValue": True,
+                "prevValue": False,
+                "propertyName": "currentValue",
             },
         },
     )
     node.receive_event(event)
     await hass.async_block_till_done()
-    state = hass.states.get(HSM200_V1_ENTITY)
-    assert state.state == STATE_UNKNOWN
+
+    state = hass.states.get(ZDB5100_ENTITY)
+    assert state.state == STATE_ON
 
     client.async_send_command.reset_mock()
 
-    # Assert that call fails if attribute is added to service call
+    # Set the brightness to 128. This should be encoded in the color value
     await hass.services.async_call(
         LIGHT_DOMAIN,
         SERVICE_TURN_ON,
-        {ATTR_ENTITY_ID: HSM200_V1_ENTITY, ATTR_RGBW_COLOR: (255, 76, 255, 0)},
+        {ATTR_ENTITY_ID: ZDB5100_ENTITY, ATTR_BRIGHTNESS: 128},
         blocking=True,
     )
-    assert len(client.async_send_command.call_args_list) == 1
+    assert len(client.async_send_command.call_args_list) == 2
     args = client.async_send_command.call_args_list[0][0][0]
     assert args["command"] == "node.set_value"
     assert args["nodeId"] == node.node_id
     assert args["valueId"] == {
         "commandClass": 51,
-        "endpoint": 0,
+        "endpoint": 1,
         "property": "targetColor",
     }
-    assert args["value"] == {"red": 255, "green": 76, "blue": 255}
-
-
-async def test_black_is_off_zdb5100(
-    hass: HomeAssistant, client, logic_group_zdb5100, integration
-) -> None:
-    """Test the black is off light entity."""
-    node = logic_group_zdb5100
-    state = hass.states.get(ZDB5100_ENTITY)
-    assert state.state == STATE_OFF
+    assert args["value"] == {
+        "red": 0,
+        "green": 128,
+        "blue": 0,
+    }
 
-    # Attempt to turn on the light and ensure it defaults to white
-    await hass.services.async_call(
-        LIGHT_DOMAIN,
-        SERVICE_TURN_ON,
-        {ATTR_ENTITY_ID: ZDB5100_ENTITY},
-        blocking=True,
-    )
-    assert len(client.async_send_command.call_args_list) == 1
-    args = client.async_send_command.call_args_list[0][0][0]
+    args = client.async_send_command.call_args_list[1][0][0]
     assert args["command"] == "node.set_value"
     assert args["nodeId"] == node.node_id
     assert args["valueId"] == {
-        "commandClass": 51,
+        "commandClass": 37,
         "endpoint": 1,
-        "property": "targetColor",
+        "property": "targetValue",
     }
-    assert args["value"] == {"red": 255, "green": 255, "blue": 255}
+    assert args["value"] is True
 
     client.async_send_command.reset_mock()
 
-    # Force the light to turn off
+    # Turn the light on and set it to green (50% brightness)
```

Comment:
Is there an easier way to reset the call args, but preserve the current mock state?
This seems unnecessarily repetitive.

Reply:
If you just want to test the service call you can just remove the receive of the event. Is that what you mean?

Reply:
The sent commands depend on the current state (brightness / color) of the light, so having the correct state before calling the service is necessary.

These assertions depend on how many calls were done before, which introduces coupling between the sub-tests
```py
    assert len(client.async_send_command.call_args_list) == 1
    args = client.async_send_command.call_args_list[0][0][0]
```

so I'm looking for a way to reset the call history, but not the entire mock state.

Reply:
You can use `reset_mock` on the mock attribute that you want to reset.

Reply:
Ok so just `client.async_send_command.call_args_list.reset_mock()`?

Reply:
No, `client.async_send_command.reset_mock()`.